packages/account/src/Components/poinc/poinc-file-uploader-container/index.js

```diff
@@ -0,0 +1,3 @@
+import PoincFileUploaderContainer from './poinc-file-uploader-container';
+
+export default PoincFileUploaderContainer;
```

Comment:
You can keep the name the same but change the file directory by removing `poinc-`
```suggestion
import PoincFileUploaderContainer from './file-uploader-container';

export default PoincFileUploaderContainer;
```

Reply:
also completed, thanks